src/main/scala/cromwell/Main.scala

```diff
@@ -68,7 +67,10 @@ object Main extends App {
         case _ => throw new RuntimeException("Expecting a JSON object")
       }
 
-      val workflowManagerSystem = new WorkflowManagerSystem {}
+      lazy val realDataAccess = new RealDataAccess
```

Comment:
@kshakir if I understand correctly now `RealDataAccess` can be removed, is that correct?